example: Makefile broken

`
make run
GOROOT=~/gowasm GOARCH=wasm GOOS=js ~/gowasm/bin/go build -o example.wasm ./client
go build -o server-app .
can't load package: package github.com/factorapp/factor/examples: no Go files in /Users/bketelsen/go/src/github.com/factorapp/factor/examples
make: *** [server-app] Error 1
`

--- a/examples/Makefile
+++ b/examples/Makefile
@@ -4,7 +4,7 @@ wasm:
 
 .PHONY: server-app
 server-app:
-	go build -o server-app .
+	go build -o server-app ./server
 
 .PHONY: run
 run: wasm server-app

